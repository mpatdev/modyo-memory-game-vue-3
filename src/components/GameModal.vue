<script>
</script>

<template>
  <div class="game-modal">
    <div class="game-modal__overlay" />
    <div class="game-modal__body" />
  </div>
</template>