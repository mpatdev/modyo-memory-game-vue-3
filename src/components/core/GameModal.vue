<template>
  <div class="game-modal ">
    <div class="game-modal__overlay" />
    <div class="game-modal__body ">
      <slot />
    </div>
  </div>
</template>

<style lang="scss">
.game-modal {
  @apply fixed inset-0 flex items-center justify-center;

  &__overlay {
    @apply bg-secondary opacity-70 absolute inset-0 w-full h-full z-[1];
  }

  &__body {
    @apply bg-white p-4 md:w-[450px] w-[85%] min-h-[200px] relative z-[5] flex items-center rounded;


  }
}
</style>