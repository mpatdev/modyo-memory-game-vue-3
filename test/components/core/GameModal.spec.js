import GameModal from '@/components/core/GameModal.vue'

import snapshotTest from '@test/snapshotTest'

snapshotTest(GameModal, 'GameModal')