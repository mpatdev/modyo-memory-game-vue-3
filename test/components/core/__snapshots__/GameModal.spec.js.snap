// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GameModal > Check Snapshot 1`] = `
"<div class=\\"game-modal\\">
  <div class=\\"game-modal__overlay\\"></div>
  <div class=\\"game-modal__body\\"></div>
</div>"
`;
